src/core/PlayArea.ttslua

```diff
@@ -8,7 +8,7 @@ DEBUG = false
 -- we use this to turn off collision handling until onLoad() is complete
 COLLISION_ENABLED = false
 
-local COUNTER = getObjectFromGUID('f182ee')
+local investigatorCounter = getObjectFromGUID('f182ee')
```

Comment:
should move this to "onLoad" to avoid issues with different load orders

Reply:
Bigger refactoring of this coming later, just trying to do a minimal disruption change (apart from the name fix, which I couldn't help myself on).

Reply:
Eh, take that back, I'll fix it.  Change incoming.

Reply:
Done.